target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -281,7 +281,8 @@ ath79_setup_interfaces()
 	tplink,tl-wdr3600-v1|\
 	tplink,tl-wdr4300-v1|\
 	tplink,tl-wdr4300-v1-il|\
-	tplink,tl-wdr4310-v1)
+	tplink,tl-wdr4310-v1|\
+	mercury,mw4530r-v1)
```

Comment:
Did you omit the WAN MAC address setup in the same file below intentionally?

Reply:
The node should be sorted alphabetically internally, so mercury as the first member, and then the whole node should be moved based on sorting of the first entry.